### DISCUSSION TITLE
无法按需加载antd样式

### DISCUSSION THREAD

**Discussion by Slebee** [umijs/umi#5903] [Q&A]
你好，我这里是使用dumi创建了一个组件包a，包a里使用了其他包 https://github.com/alibaba/formily
formily里引用了antd，然后我在umi3项目里安装包a无法加载出对应的antd组件样式，我试了babel-plugin-import 按需a包、formily 以及设置nodeModulesTransform  exclude了a包都无法加载出antd的样式，请问这个要怎么处理

**Comment by xiaohuoni** [comment]
dumi 中创建组件的时候，需要引入用到的 antd 样式。在组件中不要使用 CSS  Module，就不会有样式缺失的问题了。

**Reply by Slebee** [reply]
这样使用一些antd样式貌似会被打包两次的样子

**Comment by sorrycc** [comment]
nodeModulesTransform.exclude 需要包含 a 和 formily，因为 antd 是 formly 引入的。

**Reply by Slebee** [reply]
nodeModulesTransform: {
    type: 'none',
    exclude: [a, '@formily', '@formily/antd-components', '@formily/antd'],
  },

这样写样式还是加载不出来
